app/code/Magento/MediaGallery/Model/ResourceModel/Keyword/SaveAssetLinks.php

```diff
@@ -5,18 +5,17 @@
  */
 declare(strict_types=1);
 
-namespace Magento\MediaGallery\Model\Keyword\Command;
+namespace Magento\MediaGallery\Model\ResourceModel\Keyword;
 
-use Magento\MediaGalleryApi\Api\Data\KeywordInterface;
-use Magento\MediaGalleryApi\Model\Keyword\Command\SaveAssetLinksInterface;
 use Magento\Framework\App\ResourceConnection;
 use Magento\Framework\DB\Adapter\AdapterInterface;
 use Magento\Framework\DB\Adapter\Pdo\Mysql;
 use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\MediaGalleryApi\Api\Data\KeywordInterface;
 use Psr\Log\LoggerInterface;
 
 /**
- * Class SaveAssetLinks
+ * Save links between asset and keyword to media_gallery_asset_keyword table
```

Comment:
Rename the class, create a link DTO

Reply:
This is an internal class, not an API service